apps/cowswap-frontend/src/modules/limitOrders/services/safeBundleFlow/index.ts

```diff
@@ -131,6 +134,7 @@ export async function safeBundleFlow(
     }
 
     const safeTx = await safeAppsSdk.txs.send({ txs: safeTransactionData })
+    cowEventEmitter.emit(CowEvents.ON_POSTED_ORDER, { orderUid: orderId, chainId })
```

Comment:
Using react context we have to go through lots of hooks just to fire an event.
I think would be easier to make `cowEventEmitter` a global const.

Reply:
Done in #3838